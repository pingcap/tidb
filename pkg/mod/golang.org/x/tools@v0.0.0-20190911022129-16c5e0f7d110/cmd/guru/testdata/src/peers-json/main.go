package main

// Tests of channel 'peers' query, -format=json.
// See go.tools/guru/guru_test.go for explanation.
// See peers-json.golden for expected query results.

func main() {
	chA := make(chan *int)
	<-chA
	select {
	case <-chA: // @peers peer-recv-chA "<-"
	}
}
