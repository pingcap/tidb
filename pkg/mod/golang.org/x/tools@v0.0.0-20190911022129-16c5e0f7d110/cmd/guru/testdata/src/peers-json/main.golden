-------- @peers peer-recv-chA --------
{
	"pos": "testdata/src/peers-json/main.go:11:7",
	"type": "chan *int",
	"allocs": [
		"testdata/src/peers-json/main.go:8:13"
	],
	"receives": [
		"testdata/src/peers-json/main.go:9:2",
		"testdata/src/peers-json/main.go:11:7"
	]
}
